### PR TITLE
DOMA-7004 analytics v3 charts detail view

### PR DESCRIPTION
**Pull request by sitozzz**
API changes
- [x] Add ability to retrieve certain data entities from `GetOverviewDashboardService`
- [x] Add property and executor filters

UI changes
- [x] Add property global filter
- [x] Add detail view with own filters, certain data request and table data display
- [x] Add hint with receipt formats for payment chart
- [x] Add new ticket status to the top card
- [x] Add property units count to the top card 

image
image
image

### CODE REVIEW

File: apps/condo/domains/analytics/utils/services/dataLoaders/property.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+            where: {
+                ...where,
+                deletedAt: null,
+            },
+        })
+
+        const propertyIds = await propertyUnitDataLoader.load()
+        const result = await propertyUnitDataLoader.loadAggregate('SUM("unitsCount")', propertyIds.map(({ id }) => id))
+
+        return { sum: new Big(result.sum || 0).toFixed(0) }
```
Comment (dkoviazin) [inline apps/condo/domains/analytics/utils/services/dataLoaders/property.js:20]: Big is for decimal

File: apps/condo/domains/analytics/components/charts/payment/PaymentByPropertyChart.tsx [hunk 2]
```diff
@@ -1,36 +1,118 @@
...
+            })
+        }
+    }, [dateRange, isOpen, loadPaymentData, organizationId, propertyIds])
 
     const dataMapper = useMemo(() => PaymentByPropertyDataMapper(PaidTitle), [PaidTitle])
+    const chart = useMemo(() => (
+        <CustomChartView
+            viewMode='pie'
+            mapperInstance={dataMapper}
+            chartConfig={{ chartOptions: { height: 300 }, animationEnabled: true }}
```
Comment (dkoviazin) [inline apps/condo/domains/analytics/components/charts/payment/PaymentByPropertyChart.tsx:66]: looks like constant (height: 300)

**Review by dkoviazin — APPROVED** [review]

File: apps/condo/domains/analytics/components/charts/resident/ResidentByPropertyChart.tsx [hunk 3]
```diff
@@ -1,36 +1,116 @@
...
+    const [localData, setLocalData] = useState([])
+
+    const { values: propertyIds, SearchInput } = usePropertyFilter({ organizationId })
+    const { dateRange } = useDateRangeFilter()
+    const { open, isOpen, PopupChartView } = useDetailChartView({ title: ChartTitle })
+    const [loadResidentData, { loading }] = useLazyQuery(GET_OVERVIEW_DASHBOARD_MUTATION, {
+        onCompleted: (response) => {
+            setLocalData(get(response, 'result.overview.resident.residents', []))
+        },
+        onError: error => {console.log(error)},
```
Comment (AleX83Xpert) [inline apps/condo/domains/analytics/components/charts/resident/ResidentByPropertyChart.tsx:38]: Are we will get empty data if there is an error? Is there an ability to display "oops, something wrong"?

File: apps/condo/domains/analytics/components/charts/payment/PaymentByPropertyChart.tsx [hunk 4]
```diff
@@ -1,36 +1,118 @@
...
-                <CustomChartView
-                    viewMode='pie'
-                    mapperInstance={dataMapper}
-                    chartConfig={{ chartOptions: { height: 300 }, animationEnabled: true }}
-                    data={data}
-                />
-            </Col>
-        </Row>
+        <>
+            <Row gutter={[0, 16]}>
```
Comment (AleX83Xpert) [inline apps/condo/domains/analytics/components/charts/payment/PaymentByPropertyChart.tsx:80]: It's good to move gutters values into constants

File: apps/condo/domains/analytics/components/charts/ticket/TicketByCategoryChart.tsx [hunk 5]
```diff
@@ -1,34 +1,112 @@
...
+import type { TicketChartCardType } from './dataMappers'
 
-const TicketByCategoryChart: ITicketChartCard = ({ data }) => {
+const TicketByCategoryChart: TicketChartCardType = ({ data, organizationId }) => {
     const intl = useIntl()
     const TicketTitle = intl.formatMessage({ id: 'global.section.tickets' })
     const TicketsByCategory = intl.formatMessage({ id: 'pages.condo.analytics.TicketAnalyticsPage.groupByFilter.Category' })
+    const CategoryClassifierTitle = intl.formatMessage({ id: 'global.category' })
+
+    const title = TicketTitle + ' ' + TicketsByCategory.toLowerCase()
```
Comment (AleX83Xpert) [inline apps/condo/domains/analytics/components/charts/ticket/TicketByCategoryChart.tsx:27]:
```
`
```
 is better for concatenating of strings

**Review by AleX83Xpert — APPROVED** [review]

File: apps/condo/domains/analytics/components/charts/resident/ResidentByPropertyChart.tsx (hunk 3 above)
Comment (sitozzz) [reply apps/condo/domains/analytics/components/charts/resident/ResidentByPropertyChart.tsx:38]: Yes, I'll think about it

**Pull request merged**